Source/SwiftLintFramework/Rules/NimbleOperatorRule.swift

```diff
@@ -39,9 +39,20 @@ public struct NimbleOperatorRule: ConfigurationProviderRule, OptInRule {
             "↓expect(10).to(beLessThanOrEqualTo(10))\n",
             "↓expect(x).to(beIdenticalTo(x))\n",
             "expect(10) > 2\n ↓expect(10).to(beGreaterThan(2))\n"
+        ],
+        corrections: [
+            "↓expect(seagull.squawk).toNot(equal(\"Hi\"))\n": "expect(seagull.squawk) != \"Hi\"\n",
+            "↓expect(\"Hi!\").to(equal(\"Hi!\"))\n": "expect(\"Hi!\") == \"Hi!\"\n",
+            "↓expect(12).toNot(equal(10))\n": "expect(12) != 10\n",
+            "↓expect(value1).to(equal(value2))\n": "expect(value1) == value2\n",
+            "↓expect(value1).to(equal(10))\n": "expect(value1) == 10\n",
+            "↓expect(10).to(beGreaterThan(8))\n": "expect(10) > 8\n",
+            "↓expect(10).to(beGreaterThanOrEqualTo(10))\n": "expect(10) >= 10\n",
+            "↓expect(10).to(beLessThan(11))\n": "expect(10) < 11\n",
+            "↓expect(10).to(beLessThanOrEqualTo(10))\n": "expect(10) <= 10\n",
+            "↓expect(x).to(beIdenticalTo(x))\n": "expect(x) === x\n"
         ]
     )
-
```

Comment:
probably there were empty spaces in the line (Xcode adds them by default)

Make sure trimming is enabled:

<img width="739" alt="captura de tela 2017-01-03 as 16 26 53" src="https://cloud.githubusercontent.com/assets/833072/21618318/7920e4d2-d1d1-11e6-92a2-2f3a9ad71987.png">